.gitmodules

```diff
@@ -7,15 +7,3 @@
 [submodule "submodules/adol-c"]
 	path = submodules/adol-c
 	url = https://gitlab.com/adol-c/adol-c
```

Comment:
The DiffSharp dependencies are now downloaded via NuGet (to the builds directory, I believe) because they weren't working as submodules.

I put the ADiMat files in because it doesn't seem to be available through git to submodule, or through any other package manager. I suppose we could try getting CMake to download and unzip it, but I'm not sure how easy that would be?